core/src/main/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -337,6 +345,7 @@ public void readFrom(StreamInput in) throws IOException {
         @Override
         public void writeTo(StreamOutput out) throws IOException {
             out.writeVInt(availableProcessors);
+            out.writeVInt(allocatedProcessors);
```

Comment:
@bleskes thanks a lot. 

Hmm, I'm back porting this change to 2.1 and 2.x, but I don't see any conflict here. So could I ask what should I watch out here.